truss/patch/calc_patch.py

```diff
@@ -210,25 +215,21 @@ def _calc_python_requirements_patches(
     differences found.
     """
     patches = []
-    prev_reqs = _parsed_reqs_by_name(prev_config.requirements)
+    prev_reqs = reqs_by_name(prev_config.requirements)
```

Comment:
should we add a quick unit test for this fix?

Reply:
Add `test_requirement_name_identifier.py`